src/target/stm32f1.c

```diff
@@ -357,17 +357,25 @@ static int stm32f1_flash_unlock(target *t, uint32_t bank_offset)
 	return 0;
 }
 
-static bool stm32f1_flash_busy_wait(target *t, uint32_t bank_offset)
+static void stm32f1_flash_clear_eop(target *t, uint32_t bank_offset)
```

Comment:
Please add `const` to both parameters as they're not modified by the function. Given how short this is, it would be ideal to mark the function inline too as in:
`static inline void stm32f1_flash_clear_eop(target *const t, const uint32_t bank_offset)`

Reply:
Marking the "target *t" as const would require also to change the target_mem_read32(), which has no const marker. That would then require to change a lot more files, which should be a separate PR I think.

Reply:
There are two kinds of const. The target being const, the pointer to the target being const. We're asking you to do the latter, no changes to target_mem_read32() needed - `target *const t`